src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -65,6 +74,53 @@ public List<ExtensionRestHandler> getExtensionRestHandlers() {
             );
     }
 
+    @Override
+    public Collection<Object> createComponents(ExtensionsRunner runner) {
+
+        SDKRestClient sdkRestClient = getRestClient();
+        SDKClusterService sdkClusterService = runner.getSdkClusterService();
+        Settings environmentSettings = runner.getEnvironmentSettings();
+        NamedXContentRegistry xContentRegistry = runner.getNamedXContentRegistry().getRegistry();
+        ThreadPool threadPool = runner.getThreadPool();
+
+        JvmService jvmService = new JvmService(environmentSettings);
```

Comment:
This probably needs a TODO and an issue to make sure we're tracking the OpenSearch JVM here when we want to use the circuit breaker.